src/sugar3/activity/activity.py

```diff
@@ -1212,7 +1317,7 @@ def __get_filters_cb(self):
         }
         filter_dict = dbus.Dictionary(filters, signature='sv')
         logging.debug('__get_filters_cb %r' % dbus.Array([filter_dict],
-                      signature='a{sv}'))
+                                                         signature='a{sv}'))
```

Comment:
(FWIW I don't mind including such cosmetic changes with Python, because the language has whitespace requirements, and nonconformance with pep8 is unwise :)